### DISCUSSION TITLE
Documentation for using pybricks from Visual Studio Code on Spike Prime Hub?

### DISCUSSION THREAD

**Discussion by jaguilar** [pybricks/support#1211] [Q&A] [answered]
Hey all, is there a page that discusses how to use Visual Studio Code with the Spike Prime hub? Thanks!

(To be clear, I'm interested in developing my code using Visual Studio Code and pybricks, then uploading it to the Spike Prime hub for execution.)

**Comment by dlech** [comment, marked as answer]
https://pybricks.com/projects/tutorials/dev/tools/vscode/

**Reply by jaguilar** [reply]
You are a hero. Thank you for everything you've done!

**Comment by FLL-Team-24277** [comment]
I've also put some step-by-step instructions, geared toward FIRST Lego League teams here: https://github.com/FLL-Team-24277/Master-Program-Fall-2023

**Comment by AnandSingh** [comment]
I created an Pybricks Runner, a VSCode extension that lets you effortlessly program LEGO SPIKE Prime, EV3, and other Pybricks-compatible bricks directly within Visual Studio Code!

Check out the extension and installation guide here:
https://github.com/AnandSingh/pybricks-runner-vscode

Your feedback and contributions are warmly welcome! Feel free to open issues or submit pull requests.
